### ISSUE TITLE
duplicate device group check is overly aggressive

### ISSUE THREAD

**Issue by aerickson** [bclary/mozilla-bitbar-devicepool#119]
From discussions with Sakari, our duplicate device group check is too aggressive. We just need to ensure that the user running devicepool doesn't have duplicates, other users can have device groups with the same name.

```
May 08 18:58:35 bitbar-devicepool-0 bash[28359]: Traceback (most recent call last):
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:   File "/home/bitbar/mozilla-bitbar-devicepool/mozilla_bitbar_devicepool/main.py", line 174, in <module>
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:     main()
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:   File "/home/bitbar/mozilla-bitbar-devicepool/mozilla_bitbar_devicepool/main.py", line 171, in main
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:     args.func(args)
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:   File "/home/bitbar/mozilla-bitbar-devicepool/mozilla_bitbar_devicepool/main.py", line 54, in test_run_manager
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:     configuration.configure(bitbar_configpath, filespath=args.files, update_bitbar=args.update_bitbar)
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:   File "/home/bitbar/mozilla-bitbar-devicepool/mozilla_bitbar_devicepool/configuration.py", line 94, in configure
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:     configure_device_groups(update_bitbar=update_bitbar)
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:   File "/home/bitbar/mozilla-bitbar-devicepool/mozilla_bitbar_devicepool/configuration.py", line 166, in configure_device_groups
May 08 18:58:35 bitbar-devicepool-0 bash[28359]:     raise Exception('device group {} has {} duplicates'.format(device_group_name, len(bitbar_device_groups) - 1))
May 08 18:58:35 bitbar-devicepool-0 bash[28359]: Exception: device group motog4-docker-builder-2 has 1 duplicates
```

example query to limit to user:

`curl  --silent  --header 'Content-Type: application/json' -u <APIKEY>: https://mozilla.testdroid.com/cloud/api/v2/device-groups\?limit\=0 | jq  '.data[] | select(.displayName=="motog4-docker-builder-2" and .userEmail=="aerickson@mozilla.com(opens in new tab)")'`

**Comment by aerickson** [comment]
https://mozilla.testdroid.com/cloud/api/v2/me/device-groups seems like the best as it does filtering for us.

**Comment by aerickson** [comment]
Fixed by #120.
